cmd/soroban-rpc/internal/config/options.go

```diff
@@ -271,6 +271,69 @@ func (cfg *Config) options() ConfigOptions {
 			DefaultValue: uint(runtime.NumCPU()),
 			Validate:     positive,
 		},
+		{
+			Name:         "request-backlog-global-queue-limit",
+			Usage:        "Maximum number of outstanding requests",
+			ConfigKey:    &cfg.RequestBacklogGlobalQueueLimit,
+			DefaultValue: uint(1000),
```

Comment:
is there any room for larger default as it's an aggregate of all requests, like 5000 etc?

Reply:
I'm definitely open to modify these baked-in constants. would it be ok if I'll modify that number in the follow-up PR ?

Reply:
All these defaults, from my perspective, are a starting point values that we could adjust over time.

Reply:
yes, sounds good, I was going to ask if any prior benchmarking was referenced for these initial defaults, and understand this is just first iteration, thanks!